docker-compose.override.example.yml

```diff
@@ -4,17 +4,42 @@ services:
 
   app:
     environment:
-      - API_URL=<change-me>
-      - AUTH0_SECRET=<change-me>
-      - SLACK_WEBHOOK_URL=<change-me>
-      - JWT_SECRET=<change-me>
-      - BOT_TOKEN=<change-me>
-      # SMTP
-      - SMTP_HOST=smtp.host.com
-      - SMTP_PORT=465
-      - SMTP_USER=user@host.com
-      - SMTP_PASS=pass
-      - SMTP_FROM=User <user@host.com>
+      #
+      # Do not reorder list.
+      # Do not change a number of position.
+      # Do not override defaults.
+      # Adds params to "docker-compose.yml",
+      # to ".override.yml", ".override.example.yml".
+      # Change <change-me>.
+      #
+      # 1. Persistence
+      # - PORT=80 # 1.1
+      # - NODE_PATH=/opt/app # 1.2
+      # - NODE_ENV=development # 1.3
+      # 2. Auth0
+      - AUTH0_SECRET=nCjN8jyLJTjaWgSJ2BByrRyD2m5pQAj9PPSFmlaRPEFj7dIi4ZS5hjw7tY56MJpR # 2.1
+      - AUTH0_DOMAIN=foretold.auth0.com # 2.2
+      - AUTH0_CLIENT_ID=Kllgh8D8oBIXBM8W6Rkn1fQem3nNBS3U # 2.3
+      # 3. Main
+      - API_URL=https://foretold.io/graphql # 3.1
+      - SERVER_URL=https://foretold.io # 3.1.1
+      # - CDF_COMBINE_SIZE=1000 # 3.2
+      # - AGGR_SCHEDULE=0 * * * * # 3.3
+      - SLACK_WEBHOOK_URL=<change-me> # 3.4
+      - BOT_TOKEN=Ud3M9CThGexTVWO21DCD3VKqKO7Bqkzhk9dPWlF71h18ILZREQd7zlgJBggApoQ5 # 3.5
+      # - JWT_ISSUER=Foretold # 3.6
+      - JWT_SECRET=oDtWHuh0r44y6YSkU0886fVe5br7Thr6 # 3.7
+      # 4. SMTP
+      - SMTP_HOST=email-smtp.us-east-1.amazonaws.com # 4.1
+      # - SMTP_PORT=465 # 4.2
+      - SMTP_USER=someuser # 4.3
+      - SMTP_PASS=somepassword # 4.4
+      - SMTP_FROM=UserName <userbox@host.com> # 4.5
+      # 5. GitHub. You must be owner of the repo.
+      - GITHUB_PERSONAL_ACCESS_TOKEN=K3Gyw6JjxtQyu82iAbzNQ9GDozBESNNVKjtOboW6 # 5.1
```

Comment:
Is this a private token? Is it ok for it to be in a public github repo?

Reply:
I'd suggest we don't have things that even look secretive here. Even if they're actually not, it could raise questions to people reading this. We could deal with these in other ways.

Reply:
I had an idea that "example" file should show particular samples of values but not just "<change-me>". When one sees string with an example of value he better understand what this environment means. Sure, there were not real tokens.

Reply:
Ah. In that case could you make them obviously fake?

Reply:
Great idea, will do

Reply:
Done.